dist/images/Dockerfile.dpdk1911

```diff
@@ -1,11 +1,9 @@
-# NOTE: At time of writing, CentOS 8 does not have dpdk-devel v19.11 in the Yum repo.
-# This package is required to build the OVS-DPDK RPMs and is available in Fedora 32.
-FROM fedora:32 as dpdk
+FROM centos:8 as dpdk
 ENV BUILD_DATE 20200731
 ENV RPM_ARCH=x86_64
 
-ENV DPDK_VERSION=19.11.1
-ENV OVS_VERSION=2.13.0
+ENV DPDK_VERSION=19.11.2
+ENV OVS_VERSION=2.13
```

Comment:
Can we upgrade to ovs 2.14 here?

Reply:
just notice the pr titile, should we use dpdk 19.11.4 here？

Reply:
Yes - sorry mistake, making that change as well as the other changes mentioned above now

Reply:
Done